src/Testing/TypeInferenceTestCase.php

```diff
@@ -390,15 +395,15 @@ public static function gatherAssertTypes(string $file): array
 		}
 
 		if (count($delayedErrors) === 0) {
-			return $asserts;
+			return $asserts; // @phpstan-ignore return.type
```

Comment:
That's not really a solution I was looking for 😊

Reply:
fixed the wrong return-type